site/docs/get-started/quickstart.md

```diff
@@ -1,61 +1,90 @@
 
-- The control plane of Mesh for Data
-- Hashicorp Vault and connector as the credentials manager 
-- ODPi Egeria lab and connectors as the data catalog
-- Open Policy Agent (OPA) and connector as the policy manager.
-- Arrow-Flight data access module for reading Parquet and CSV datasets
+```bash
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 120s
+``` 
 
-## Prepare for installing Mesh for Data
+## Install Hashicorp Vault and plugins
 
-1.  Obtain a local copy of Mesh for Data repository
-    ```bash
-    git clone https://github.com/ibm/the-mesh-for-data.git
-    ```
-1.  Change to the root directory of the repository
-    ```bash
-    cd the-mesh-for-data
-    ```
+[Hashicorp Vault](https://www.vaultproject.io/) and a [secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) plugin are used by Mesh for Data for credential management.
 
-## Install Mesh for Data
+Run the following to install vault and the plugin in development mode:
 
-1. Set the current namespace to `m4d-system`
+=== "Kubernetes" 
 
     ```bash
-    kubectl config set-context --current --namespace=m4d-system
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/values.yaml \
+        --wait --timeout 120s
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
     ```
 
-1. Run the quick install script to install the control plane elements.
+=== "OpenShift"
 
     ```bash
-    ./hack/install.sh
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "global.openshift=true" \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/values.yaml \
+        --wait --timeout 120s
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
     ```
 
-    to install on OpenShift you need to run ```WITHOUT_OPENSHIFT=false ./hack/install.sh``` instead.
+## Install control plane
 
-1. Enable the use of the [arrow flight module](https://github.com/ibm/the-mesh-for-data-flight-module)
+??? tip "Install development version"
```

Comment:
maybe change "Install development version" to "Install latest development version from github"

Reply:
Okay